math.js

```diff
@@ -0,0 +1,3 @@
+function add(a, b) {
```

Comment:
What is function in javascript?

Reply:
That is the fundamental unit of abstracting an operation